src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -144,12 +144,10 @@ protected override void OnSaveOptions(string key, Stream stream)
 
     protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
     {
-        await base.InitializeAsync(cancellationToken, progress).ConfigureAwait(true);
```

Comment:
>true [](http://example.com/codeflow?start=79&length=4)

No need to ensure we're on the captured context if the next line moves us to the main thread.